File: app/src/main/java/br/edu/ifce/engcomp/francis/radarpolitico/models/Dia.java
package br.edu.ifce.engcomp.francis.radarpolitico.models;

/**
 * Created by francisco on 24/03/16.
 */
public class Dia {
    String data;
    String frequencia;
    int numeroSessoes;

    public Dia() {
    }

    public String getData() {
        return data;
    }

    public int getNumeroSessoes() {
        return numeroSessoes;
    }

    public void setNumeroSessoes(int numeroSessoes) {
        this.numeroSessoes = numeroSessoes;
    }

    public void setData(String data) {
        this.data = data;
    }

    public String getFrequencia() {
        return frequencia;
    }

    public void setFrequencia(String frequencia) {
        this.frequencia = frequencia;
    }
}

File: app/src/main/kotlin/br/edu/ifce/engcomp/francis/radarpolitico/models/Dia.kt
package br.edu.ifce.engcomp.francis.radarpolitico.models

import java.io.Serializable

/**
 * Created by francisco on 24/03/16.
 */
data class Dia(
    var data: String? = "",
    var frequencia: String? = "",
    var numeroSessoes: Int = 0
) : Serializable
